index.html

```diff
@@ -787,7 +787,7 @@ <h3>
           The [=MiniApp manifest's=] <code><dfn>app_id</dfn></code> member is a [=string=] that identifies the MiniApp univocally. This member is mainly used for package management, and it supports the update and release process of MiniApp versioning.
```

Comment:
The word "univocally" is used correctly here, but, as a native English speaker, it's an extremely rare word (I had to look it up). I would suggest replacing it with the much more common "uniquely" or, perhaps, univocally's synonym "unequivocally".

Reply:
Thank you @aphillips . Changed as suggested. 
I don't know where I learned that word. I've been using it for a while... thinking it was totally valid :-)  Changed also in my dictionary :-)